packages/common/src/content/_internal/ContentBusinessRules.ts

```diff
@@ -121,4 +122,9 @@ export const ContentPermissionPolicies: IPermissionPolicy[] = [
     permission: "hub:content:canRecordDownloadErrors",
     environments: ["qaext", "devext"],
   },
+  {
+    permission: "hub:content:downloads:displayErrors",
+    availability: ["alpha"],
```

Comment:
Since we're limiting to qa and dev, do we really want to alpha gate as well?

Reply:
@sonofflynn89 this is generally what we've been doing